fpga/Makefile

```diff
@@ -71,6 +85,7 @@ default: $(mcs)
 # misc. directories
 #########################################################################################
 fpga_dir := $(base_dir)/fpga/fpga-shells/$(FPGA_BRAND)
+fpga_board_script_dir := $(fpga_dir)/$(BOARD)/tcl
```

Comment:
Where is this used?

Reply:
It is originally used in [sifive/freedom](https://github.com/sifive/freedom/blob/master/common.mk). I do not remember the original intention. It can be removed since we no longer need it currently.